src/__tests__/drop.test.js

```diff
@@ -0,0 +1,69 @@
+import drop from '../drop';
+
+describe('drop', () => {
+    describe('positive tests', () => {
+        describe('common array cases', () => {
+            test('should drop the default 1 element from the beginning', () => {
```

Comment:
```suggestion
            test('should drop the first element from the beginning by default, if no parameter is provided', () => {
```

Reply:
Fixed in commit [16760ea](https://github.com/erictelkkala/COMP.SE.200-2024-2025-1/pull/16/commits/16760ea4aa0f9b588a4f9a9a1a38b0c3ef560805)